test/NuGet.Tests.Apex/NuGet.Tests.Apex/NuGetEndToEndTests/NuGetConsoleTestCase.cs

```diff
@@ -713,6 +712,84 @@ public async Task VerifyCacheFileInsideObjFolder(ProjectTemplate projectTemplate
             }
         }
 
+        [NuGetWpfTheory]
+        [MemberData(nameof(GetPackagesConfigTemplates))]
+        public async Task UpdateAllPCPackagesInPMC(ProjectTemplate projectTemplate)
+        {
+            EnsureVisualStudioHost();
+            using (var simpleTestPathContext = new SimpleTestPathContext())
+            {
+                // Arrange
+                var packageName1 = "UpdateAllPCPackage1";
+                var packageVersion1 = "1.0.0";
```

Comment:
The version strings here should probably be the version itself. 

Example:
```suggestion
                var v100 = "1.0.0";
```

Reply:
Make sense, we will update the case later.